Docker.README.md

```diff
@@ -0,0 +1,48 @@
+# Running duke-data-service in docker
```

Comment:
It's not clear why these instructions (running duke-data-service) are included in this repo.

I assume it's so that developers can get a server up-and-running to test against, but the reasoning should probably be included. And I'd suggest changing the file name too, since `Docker.README.md` isn't very descriptive. Many projects add a section to the main README for contributing or developing. Maybe link to the instructions from there?